src/renderer/atlas/common.h

```diff
@@ -377,6 +377,7 @@ namespace Microsoft::Console::Render::Atlas
     {
         u32 backgroundColor = 0;
         u32 selectionColor = 0x7fffffff;
+        u32 searchSelectionColor = 0x5fff0000;

```

Comment:
I meant to ask about this,  the blue I used probably isn't a great default,  should I try to add this as setting?

Reply:
Personally speaking, I wouldn't consider that important. Some of the most popular applications don't have configurable highlight colors, including Chromium and Word.

Reply:
It looks like this value is no longer used. Should we remove it?